book/tutorials/jupyter.md

```diff
@@ -173,19 +173,19 @@ for a git repository.
 ### Add the {{hackweek}} organization repository as a remote
 
 A remote is defined with a label and a URL. In our case and command below,
-we are using `{{hackweek}}` as the label and `{{website_url}}.git`
+we are using Hackweek as the label and {{ '`{url}.git`'.format(url=website_url) }}
 as the URL.
 
 ```{Note}
-The chosen label `{{hackweek}}` is found in many projects as the `upstream`. Here we
+The chosen label Hackweek is found in many projects as the `upstream`. Here we
 just went with a more descriptive label instead of the default value.
 ```
 
 ```shell
 # Change to the website folder, it was created with the `clone`
 cd website
 # Add the remote
-git remote add {{hackweek}} {{website_url}}.git
+git remote add Hackweek https://github.com/ICESAT-2HackWeek/website2022.git
```

Comment:
Can we use `{{website_url}}.git` here and have it render properly, or no because it's in a shell block?